src/Styleguide/Elements/Flex.tsx

```diff
@@ -1,20 +1,28 @@
 import styled from "styled-components"
-import { FlexboxProps, MarginProps } from "../StyledSystemTypes"
+
 import {
-  flexDirection,
-  alignItems,
   alignContent,
+  alignItems,
+  AlignItemsProps,
+  flexDirection,
+  FlexDirectionProps,
   justifyContent,
+  JustifyContentProps,
   space,
+  SpaceProps,
```

Comment:
@zephraph - now that we're pulling interfaces into our components / primitives lets make sure our styled-system stuff is all alpha so its clear at a glance what's in scope and what still needs types.